### PR TITLE
News comment UI as per issues #62 and #63

### PR DESCRIPTION
**Pull request by inderjeet-developer7**
// To do
- Adapt mocked data from API when implemented instead of static one's
- Open keyboard on Reply text tap and swipe action

### CODE REVIEW

**Comment by gnunicorn** [comment]
@inderjeet-developer7 please fix the remaining [style error `flutter analyze` is complaining about in the CI job](https://github.com/effektio/effektio/runs/6702195351?check_suite_focus=true#step:6:12)

File: app/lib/screens/HomeScreens/ChatScreen.dart [hunk 1]
```diff
@@ -115,7 +115,7 @@ class _ChatScreenState extends State<ChatScreen> {
     final index = _messages.indexWhere((element) => element.id == message.id);
     final updatedMessage = _messages[index].copyWith(previewData: previewData);
 
-    WidgetsBinding.instance.addPostFrameCallback((_) {
```
Comment (gtalha07) [inline app/lib/screens/HomeScreens/ChatScreen.dart:118]: Please update your flutter SDK to 3.0 as this is not the error and revert this change in new commit. This is deprecated and it isn't required to make ```WidgetBindings``` nullable now.

Comment (inderjeet-developer7) [reply app/lib/screens/HomeScreens/ChatScreen.dart:118]: okay
